drivers/hud_spm_report/app/models/hud_spm_report/generators/fy2023/measure_one.rb

```diff
@@ -4,16 +4,18 @@
 # License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
 ###
 
+# frozen_string_literal: true
+
 # HUD SPM Report Generator: Length of Time Persons Remain Homeless
 module HudSpmReport::Generators::Fy2023
   class MeasureOne < MeasureBase
     def self.question_number
-      'Measure 1'.freeze
+      'Measure 1'
     end
 
     def self.client_class
       HudSpmReport::Fy2023::Episode.
-        joins(:enrollments).preload(:enrollments)
+        joins(:enrollments).preload(:enrollments, enrollments: { enrollment: :project })
```

Comment:
I _think_ rails will just eat the first declaration, but it should be unnecessary.
```suggestion
        joins(:enrollments).preload(enrollments: { enrollment: :project })
```

Reply:
updated [here](https://github.com/greenriver/hmis-warehouse/pull/5183/commits/200b67b6f9e1ad8a08e0f4a50fde5d0bb4f3bec9)